Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3585,6 +3588,37 @@ extension BrowserViewController: UNUserNotificationCenterDelegate {
   }
 }
 
+// MARK: UIScreenshotServiceDelegate
+
+extension BrowserViewController: UIScreenshotServiceDelegate {
+  public func screenshotService(_ screenshotService: UIScreenshotService, generatePDFRepresentationWithCompletion completionHandler: @escaping (Data?, Int, CGRect) -> Void) {
+      
+    guard screenshotService.windowScene != nil,
+          presentedViewController == nil,
+          let webView = tabManager.selectedTab?.webView,
+          let url = webView.url,
```

Comment:
Need to check for `url.isAboutHomeURL` and `isAboutURL` unless you want the user to be able to screenshot NTP?
Also the same for any `InternalURL` unless they can screenshot error pages and reader-mode?

Reply:
We do not need any other check here because we are already checking If url `isWebPage`, this covers Reader Mode, NTP, Error pages.